src/mumble/Audio.h

```diff
@@ -17,6 +17,9 @@
 
 #define SAMPLE_RATE 48000
 
+// interaural delay (in samples) for a sound coming directly from the side of the head
+constexpr float INTERAURAL_DELAY = 0.00043 / (1 / static_cast< float >(SAMPLE_RATE));
```

Comment:
Missing comment on how this number comes to be. You explained it in this PR but it should also be included in the code.